### DISCUSSION TITLE
Local thumbnail for local and remote monitoring on a headless box.

### DISCUSSION THREAD

**Discussion by TheSashmo** [CESNET/UltraGrid#162] [Q&A]
I was looking at a way to enable a local preview of the transmit, as well as a way to get thumbnails on the receive.

Ideally I want to make a web controller to be able to validate signal for both audio and video.  Without being next to the device where the SDI is outputting or connected as an input. 

I don't need it in full FPS or super realtime, but I would consider a low frame rate and delayed png/jpg that I could code into a html.

I know the multiplier and maybe the capture filter preview could be used on the desktop client, but if I wanted to create a local feed via CLI that I could take snapshots and update on a local web server. 

Any suggestions on how you would do it?  Is it possible to push the capture preview to something else than the local display?

I was thinking to write a bit of code to support reading the audio measurements, and then drawing the levels in a nice animated audio bar graph too, I am not sure if there is a better way to handle that.

**Comment by MartinPulec** [comment]
> Is it possible to push the capture preview to something else than the local display?

Not really, but I think that it might not be difficult to create/tailor the existing one to dump a file in some interval.

**Comment by alatteri** [comment]
I don't know if this is any help as a starting point.  Not mine, just something I came across. 
https://github.com/ua-i2cat/ultragridwebgui

**Reply by TheSashmo** [reply]
Ahh interesting.  I need to check this out!

Thank you Alatteri!
